README.md

```diff
@@ -1 +1,247 @@
+
+- <a href="#">Live Demo</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+<!-- > Describe how a new developer could make use of your project. -->
+
+### Prerequisites
+
+To get a local copy up and running, follow these steps.
+
+In order to run this project you need:
+
+- A text editor like Microsoft Visual Studio Code.
+- A Github account.
+- Internet Connection.
+- Basic HTML, CSS and JavaScript knowledge.
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Bifabig/awesome-books.git
+```
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Run this command inside the cloned repository
+
+```sh
+npm install
+```
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+Open the index.html file in your browser
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+For styling
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+
+For JavaScript
+
+```sh
+  npx eslint .
+```
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+Github Pages
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+<!-- > Mention all of the collaborators of this project. -->
+
+👤 **Biftu Girma**
+
+- GitHub: [@Bifabig](https://github.com/Bifabig)
+- Twitter: [@biftu94](https://twitter.com/biftu94)
+- LinkedIn: [Biftu Girma](https://www.linkedin.com/in/biftu-girma-3598a8126/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Make the tasks draggable**
+
+<!-- > Mention future features. -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project:thumbsup:
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+<!-- > Give credit to everyone who inspired your codebase. -->
+
+I would like to thank Microverse and my coding partners.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] I noticed that the link to the M.I.T. license in the project is broken. The M.I.T. license is an important part of the project's licensing and it's essential that the link is working properly. kindly fix this :wink: 
![Screenshot from 2023-05-06 10-57-40](https://user-images.githubusercontent.com/88894525/236614904-2fbeb090-01c9-4177-a2a2-994cb47967ec.png)


_note: Having a broken link can cause confusion and potentially lead to legal issues, which we want to avoid. Therefore, I recommend fixing the link as soon as possible. You can try updating the link to the license or finding an alternative source for the license text._